docs/bulk_data.rst

```diff
@@ -0,0 +1,62 @@
+=========
+Bulk Data
+=========
+
+Mapping File
+===========
+
+Step Name: (must be unique)
```

Comment:
This should be wrapped in `.. code-block:: yaml` with all sub lines indented.  See the features.rst file for examples.